src/Movie-Card/Movie-Card.css

```diff
@@ -0,0 +1,75 @@
+.movie-card {
+  height: 90%;
+  width: auto;
+  padding: 0;
+  margin: 20px;
+  border-radius: 5px;
```

Comment:
```suggestion

```

Reply:
I moved this to the img, it wasn't showing b/c the image is smaller then the container.